cli/cmd/integration.go

```diff
@@ -522,6 +525,26 @@ func reflectIntegrationData(raw api.RawIntegration) [][]string {
 
 		return out
 
+	case api.DatadogChannelIntegration.String():
+
+		var iData api.DatadogChannelData
+		err := mapstructure.Decode(raw.Data, &iData)
+		if err != nil {
+			cli.Log.Debugw("unable to decode integration data",
+				"integration_type", raw.Type,
+				"raw_data", raw.Data,
+				"error", err,
+			)
+			break
+		}
+		out := [][]string{
+			[]string{"DATADOG SITE", iData.DatadogSite},
+			[]string{"DATADOG TYPE", iData.DatadogType},
```

Comment:
I think this "type" is actually the Datadog Service:

![image](https://user-images.githubusercontent.com/5712253/106493894-fc7e8280-6476-11eb-956c-ef8df30df85f.png)

```suggestion
			[]string{"DATADOG SERVICE", iData.DatadogType},
```